default.nix

```diff
@@ -23,16 +23,20 @@ let
           "-f-dev-mode"
           "--ghc-option=-optl-lm"
         ];
-        testTarget = "--log=test.log || (sleep 10 && kill $TAILPID && false)";
         preCheck = ''
-          mkdir -p dist/test
-          touch dist/test/test.log
-          tail -F dist/test/test.log &
-          export TAILPID=$!
+          minutes=0
+          limit=30               # 30 minutes for tests
+          export TESTS_RUNNING="true"
+          while [ "$TESTS_RUNNING" = "true" ]; do
+              echo -n -e " \b"
+              if [ $minutes == $limit ]; then break; fi
+              minutes=$((minutes+1))
+              sleep 60
+          done &
         '';
```

Comment:
Before merging, verify the output of failing tests is printed. Prior to the `tail -F` approach test results, including failure details, were being written to a log file whose contents weren't printed.

Reply:
@jmitchell 

> verify the output of failing tests is printed

At least locally it does.